plugins/modules/katello_sync_plan.py

```diff
@@ -143,7 +143,7 @@ def main():
 
     sync_plan = find_sync_plan(module, entity_dict['name'], entity_dict['organization'], failsafe=True)
 
-    entity_dict = sanitize_entity_dict(entity_dict, name_map)
+    entity_dict = sanitize_entity_dict(entity_dict, name_map=name_map)
```

Comment:
why this change? (besides that I am not sure I want to touch the nailgun modules at all), `sanitize_entity_dict` takes exactly two params :)

Reply:
`sed -i '...' *.py`
I can take it back. ;)

Reply:
I'd prefer not to touch these files.